geaflow/geaflow-dsl/geaflow-dsl-plan/src/main/java/com/antgroup/geaflow/dsl/schema/function/BuildInSqlFunctionTable.java

```diff
@@ -18,15 +18,7 @@
 import com.antgroup.geaflow.dsl.common.function.UDAF;
 import com.antgroup.geaflow.dsl.planner.GQLJavaTypeFactory;
 import com.antgroup.geaflow.dsl.schema.GeaFlowFunction;
-import com.antgroup.geaflow.dsl.udf.graph.AllSourceShortestPath;
-import com.antgroup.geaflow.dsl.udf.graph.ClosenessCentrality;
-import com.antgroup.geaflow.dsl.udf.graph.IncWeakConnectedComponents;
-import com.antgroup.geaflow.dsl.udf.graph.KCore;
-import com.antgroup.geaflow.dsl.udf.graph.KHop;
-import com.antgroup.geaflow.dsl.udf.graph.PageRank;
-import com.antgroup.geaflow.dsl.udf.graph.SingleSourceShortestPath;
-import com.antgroup.geaflow.dsl.udf.graph.TriangleCount;
-import com.antgroup.geaflow.dsl.udf.graph.WeakConnectedComponents;
+import com.antgroup.geaflow.dsl.udf.graph.*;
```

Comment:
It's necessary to expand the actual packages and avoid using wildcards

Reply:
fixed